app/components/autocomplete/slash_suggestion/slash_suggestion_item.js

```diff
@@ -39,44 +40,69 @@ export default class SlashSuggestionItem extends PureComponent {
         return (
             <TouchableWithFeedback
                 onPress={this.completeSuggestion}
-                style={[style.row, padding(isLandscape)]}
-                type={'opacity'}
+                style={[padding(isLandscape)]}
+                underlayColor={changeOpacity(theme.buttonBg, 0.08)}
+                type={'native'}
             >
-                <Text style={style.suggestionName}>{`${suggestion} ${hint}`}</Text>
-                <Text style={style.suggestionDescription}>{description}</Text>
+                <View style={style.container}>
+                    <View style={style.icon}>
+                        <Image
+                            style={style.iconColor}
+                            width={10}
+                            height={16}
+                            source={slashIcon}
+                        />
+                    </View>
+                    <View style={style.suggestionContainer}>
+                        <Text style={style.suggestionName}>{`${suggestion.substring(1, suggestion.length)} ${hint}`}</Text>
```

Comment:
why the substring? and you could do this

```suggestion
                        <Text style={style.suggestionName}>{`${suggestion.substr(1)} ${hint}`}</Text>
```

Reply:
The `suggestion` value comes with a `/` at the beginning, which I was asked to remove per [this comment](https://github.com/mattermost/mattermost-mobile/pull/4531#pullrequestreview-456028212)

I made sure to change it to your suggestion :)